tests/test_sql_tools.py

```diff
@@ -69,60 +67,89 @@ def test_remove_two_index(self):
         for term in test_values:
             self.assertTrue(not pd.isna(term) and term not in [None, np.nan, ''])
 
-    def test_create_locality(self):
-        """testing create_locality function by
-           recreating insert protocol for locality table, but with sqlite DB"""
+    def test_insert_table_record(self):
```

Comment:
newer version of function required a collecting event id, so had to switch to manually creating the insert statement to test the insert function.